Live/Classes/MapManager.h

```diff
@@ -15,11 +17,13 @@ class MapManager
     bool checkCollision(cocos2d::Point position);
     bool checkCollision(cocos2d::Point position, float x, float y);
     cocos2d::TMXTiledMap* getTileMap();
+    void renderPlayer(Player* player);
```

Comment:
I don't think render player is an appropriate name for this function, doesn't this just add the player to the resource layer of the map?

Reply:
gotcha, got any suggestions for a function name? addPlayerToResourceLayer?